src/github.com/getlantern/lantern-ui/dist/js/app.js

```diff
@@ -824,11 +880,36 @@ angular.module('app.services', [])
 
 'use strict';
 
-app.controller('RootCtrl', ['$scope', '$http', 'flashlightStats', function($scope, $http, flashlightStats) {
+app.controller('RootCtrl', ['$scope', '$compile', '$window', '$http', 
+               'localStorageService', 'flashlightStats', 
+               function($scope, $compile, $window, $http, localStorageService, flashlightStats) {
     // disabling for now flashlightStats.connect();
     $scope.currentModal = 'none';
 
+    $scope.loadScript = function(src) {
+        (function() { 
+            var script  = document.createElement("script")
+            script.type = "text/javascript";
+            script.src  = src;
+            script.async = true;
+            var x = document.getElementsByTagName('script')[0];
+            x.parentNode.insertBefore(script, x);
+        })();
+    };
+    $scope.loadShareScripts = function() {
+        if (!$window.twttr) {
+            // inject twitter share widget script
+            $scope.loadScript('//platform.twitter.com/widgets.js');
```

Comment:
Oh, I see what happened.  This is in the dist folder, which is generated at cross compile time.  I think valencia's dist folder is just out of date.  You can safely ignore any files in this PR that are in the dist folder instead of the app folder.